src/charm.py

```diff
@@ -353,13 +396,19 @@ def scrape_jobs(self) -> List[Dict[str, Any]]:
 
         return [job]
 
+    @property
+    def node_exporter_scrape_jobs(self) -> List[Dict[str, Any]]:
+        """Generate scrape jobs for the node exporter."""
+        job: Dict[str, Any] = {"static_configs": [{"targets": [f"{self.hostname}:9100"]}]}
```

Comment:
Should we add scheme / TLS to node-exporter as well?

Reply:
Good catch! Yes!

Reply:
Well, I'm not so sure... it's an experimental feature, look:

https://github.com/prometheus/node_exporter?tab=readme-ov-file#tls-endpoint

Reply:
I think it's marked as experimental in several grafana products, for a while now.

Reply:
Well, actually node-exporter is not a Grafana product, it's part of Prometheus project.

It's not clear to me how serious they are when they say: [_"This is experimental and might change in the future."_](https://github.com/prometheus/exporter-toolkit/blob/master/docs/web-configuration.md)

Reply:
The current image we use does not have `update-ca-certificates`. Once we create [our own rock](https://github.com/canonical/loki-k8s-operator/issues/378), we may add that package in order to have HTTPS for the node exporter.